.github/workflows/build_and_test.yml

```diff
@@ -421,144 +424,65 @@ jobs:
       run: |
         cd OpenDDS
         ./configure --optimize --no-debug --features=versioned_namespace=1 --tests --java=/usr/lib/jvm/adoptopenjdk-12-hotspot-amd64 --rapidjson --ace=$GITHUB_WORKSPACE/ACE_TAO/ACE --mpc=$GITHUB_WORKSPACE/MPC
+    - name: make OpenDDS
+      run: |
+        cd OpenDDS
+        make -j6
     - name: check build configuration
       shell: bash
       run: |
         cd OpenDDS
         . setenv.sh
         tools/scripts/show_build_config.pl
     - uses: ammaraskar/gcc-problem-matcher@0.1
-    - name: build OpenDDS
+    - name: make install OpenDDS
       shell: bash
       run: |
         cd OpenDDS
+        set -e
+        export DESTDIR=$(realpath install_temp)
+        export INSTALL_PREFIX=$(realpath install_prefix)
         . setenv.sh
-        make -j4
-    - name: build and run CMake tests
+        cd $ACE_ROOT
+        make install
+        cd $TAO_ROOT
+        make install
+        cd $GITHUB_WORKSPACE/OpenDDS
+        make install
+        cp -r $DESTDIR/$INSTALL_PREFIX .
+        cat << EOF > install_setenv.sh
+        export OPENDDS_INSTALL_PREFIX=$INSTALL_PREFIX
+        source \$OPENDDS_INSTALL_PREFIX/share/ace/ace-devel.sh
+        source \$OPENDDS_INSTALL_PREFIX/share/tao/tao-devel.sh
+        source \$OPENDDS_INSTALL_PREFIX/share/dds/dds-devel.sh
+        export PATH="\$OPENDDS_INSTALL_PREFIX/bin:\$PATH"
+        export LD_LIBRARY_PATH="\$OPENDDS_INSTALL_PREFIX/lib:\$LD_LIBRARY_PATH"
+        export MPC_ROOT=$GITHUB_WORKSPACE/MPC
+        export PERL5LIB=$(realpath bin):$(realpath $ACE_ROOT/bin)
+        EOF
+    - name: test make install
       shell: bash
       run: |
         cd OpenDDS
-        . setenv.sh
-        cd tests/cmake_integration
-        perl run_ci_tests.pl
+        cat install_setenv.sh
+        . install_setenv.sh
+        cd tests/DCPS/Messenger
+        git clean -dfx .
+        $ACE_ROOT/bin/mwc.pl -type gnuace .
+        make
+        perl run_test.pl rtps
+    - name: remove install clutter
+      shell: bash
+      run: |
+        cd OpenDDS
+        rm -rf install_temp install_prefix install_setenv.sh
     - name: gitclean
       shell: bash
       run: |
         touch output.txt
         cd OpenDDS
         git clean -nd -e ext | tee ../output.txt
         if [ -s ../output.txt ]; then exit 1; fi
-    - name: create OpenDDS tar.xz artifact
-      shell: bash
-      run: |
-        cd OpenDDS
-        rm -rf ACE_TAO
-        find . -iname "*\.o" | xargs rm
-        tar cvf ../${{ github.job }}.tar setenv.sh
-        git clean -xdfn | cut -d ' ' -f 3- | xargs tar uvf ../${{ github.job }}.tar
-        xz -3 ../${{ github.job }}.tar
-    - name: upload OpenDDS artifact
-      uses: actions/upload-artifact@v2
-      with:
-        name: ${{ github.job }}_artifact
-        path: ${{ github.job }}.tar.xz
-
-  test_u18_o1d0v1_xer0_j12_2:
```

Comment:
Are we getting rid of the whole test job for this one? Do we have comparable coverage elsewhere? If we are dropping it, should we at least add a java test using the install path?

Reply:
Aside from make install, this is a pretty unremarkable build, so I didn't feel that the full test suite was necessary for it.  I figured that if one or a few tests pass with make install then the rests probably are fine as well.

I am open to readding the full run of tests if you think it is necessary.